docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       - redis
 
   postgres:
-    container_name: postgres
+    container_name: postgres-shopify-starter
```

Comment:
if you want to use unique container names make them consistent like with the app "shopify-starter-app"